internal/args/unmarshal.go

```diff
@@ -315,9 +335,13 @@ func isUnmarshalableValue(dest reflect.Value) bool {
 
 	_, isUnmarshaller := interface_.(Unmarshaller)
 	_, hasUnmarshalFunc := unmarshalFuncs[dest.Type()]
-	_, isScalar := scalarKinds[dest.Kind()]
 
-	return isUnmarshaller || hasUnmarshalFunc || isScalar
+	return isUnmarshaller || hasUnmarshalFunc || isScalar(dest)
```

Comment:
why create a function for `isScalar` ans not the 2 others ?

Reply:
oops, removing this leftover